src/plugins/widgets/trello/SettingsInput.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import "./Trello.sass";
+
+interface SettingsProps {
+    add_id_name_pair: (id: string, name: string) => void;
+};
+
+export const SettingsInput: React.FC<SettingsProps> = ({ add_id_name_pair }) => {
+    const [listID, setListID] = React.useState("");
+    const [name, setName] = React.useState("");
+    
+    const handle_submit = (e: any)  => {
+        e.preventDefault();
+        if (listID === "" || name === "") {
+            return;
+        }
+        
+        // Add list ID name pair
+        add_id_name_pair(listID, name);
+        setListID("");
+        setName("");
+    };
+
+    return (
+        <form onSubmit={handle_submit} className="submission-form">
```

Comment:
Are you using the class name for styling here?

Reply:
No I'm not.  I'll remove it.